src/authorization/authorization.ts

```diff
@@ -878,10 +920,16 @@ export function initialize(
         };
       };
       try {
-        merge = getMergeDefaultValue(merge);
-        const result = await tryMergeUser(userId, false, merge);
+        disableMergeAndEventReplay = getMergeDefaultValue(
```

Comment:
Can we remove this logic for merging the user in setEmail and setUserId? We won't merge the user within the setUserID function. We will move the merge specific logic its own function.

The param should be disableEventReplay which will only affect whether the event gets replayed when going from an anonymous user to a known user.

Reply:
yes removed and updated param